configs/rbac_cel.yaml

```diff
@@ -0,0 +1,157 @@
+# RBAC with policies based on CEL expressions
+# curl -v "localhost:7777" will proxy to localhost:9901, the admin page
+
+admin:
+  address:
+    socket_address:
+      protocol: TCP
+      address: 127.0.0.1
+      port_value: 9901
+static_resources:
+  listeners:
+  - name: localhost-listener
+    address:
+      socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 7777
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          access_log:
+          - name: envoy.access_loggers.stdout
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StdoutAccessLog
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match:
+                  prefix: "/"
+                route:
+                  cluster: admin_cluster
+              request_headers_to_add:
+                - header:
+                    key: server
+                    value: envoy
+                  append: true
+          http_filters:
+          - name: rbac-filter-with-policies-based-on-cel-expressions
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC
+              rules:
+                action: DENY
+                policies:
+#                  "deny-admin-clusters-endpoint-policy":
```

Comment:
I wouldn't have expected to see commented-out content in these files. Can you say more about this?